### PR TITLE
Add the iea15mw power curve for uniform inflow

### PR DESCRIPTION
**Pull request by psakievich**
Adding the power curve for the iea15mw that was developed in the https://github.com/gantech/hfm_fy23_q1/
repository as part of the WETO-HFM milestones for improving and hardening exawind's FSI capabilities

### CODE REVIEW

File: iea15mw/power_curve/aprepro/eagle_aprepro.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{SLURM_ARGS = '
+#SBATCH -A hfm
+#SBATCH -t 48:00:0'}
+
+{MACHINE_NAME="eagle"}
+{NALU_RANK_PERCENTAGE=75}
+{MESH="/projects/hfm/kmoore/meshes/iea15mw_3blade_volume_v0.2.exo"}
```
Comment (kevmoor) [inline iea15mw/power_curve/aprepro/eagle_aprepro.txt:7]: Should we put the mesh used in a better location and update this path?

File: iea15mw/power_curve/envs/eagle_setup_env.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+source /nopt/nrel/ecom/hpacf/env.sh # if you aren't already doing this
+module use /projects/hfm/psakiev/spack-manager/modules
+module load exawind-master/2024-02-20
```
Comment (kevmoor) [inline iea15mw/power_curve/envs/eagle_setup_env.sh:3]: Should we make a note about this is subject to change?

File: iea15mw/power_curve/aprepro/eagle_aprepro.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+{DLL="/lustre/eaglefs/projects/hfm/psakiev/spack-manager/environments/2024-01-16/opt/linux-rhel7-skylake_avx512/intel-20.0.2/rosco-2.8.0-poo5twrrbzusvzmgb4xy7sc3bewn75to/lib/libdiscon.so"}
```
Comment (kevmoor) [inline iea15mw/power_curve/aprepro/eagle_aprepro.txt:8]: Is this adequate for this stage?

File: iea15mw/power_curve/submission_scripts/eagle-hpc-submit-power-curve.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+wind_speeds=(5.0 6.0 7.0 8.0 9.0 10.0 10.59 12.0 14.0 17.0 20.0 23.0 25.0)
+for w in "${wind_speeds[@]}"; do
+  echo "Setting up directory for wind speed $w"
+  ./setup_case.sh -m=eagle -e="kevmoor@sandia.gov" -w="$w" -s=0 -p=1 -c=1 -n=90 -l=80.0 -d=/scratch/kmoore
```
Comment (kevmoor) [inline iea15mw/power_curve/submission_scripts/eagle-hpc-submit-power-curve.sh]: Do we need to add a note to drop to your own scratch folder or is it fine as is, acting as an example?

Comment (psakievich) [reply iea15mw/power_curve/submission_scripts/eagle-hpc-submit-power-curve.sh]: no we need to make these user agnostic.  IMO you should clone this onto scratch and run it there if you want the results in scratch.  We also need to remove your email

File: iea15mw/power_curve/aprepro/eagle_aprepro.txt (hunk 1 above)
Comment (psakievich) [reply iea15mw/power_curve/aprepro/eagle_aprepro.txt:7]: I am not too concerned about this.  Consistency is the most important thing to me. If we change the mesh and publish results we need to update the repo.

File: iea15mw/power_curve/envs/eagle_setup_env.sh (hunk 2 above)
Comment (psakievich) [reply iea15mw/power_curve/envs/eagle_setup_env.sh:3]: I don't intend to change it. Ideally we should keep these up to date along the same lines ad the comment on the mesh.

File: iea15mw/power_curve/aprepro/eagle_aprepro.txt (hunk 3 above)
Comment (psakievich) [reply iea15mw/power_curve/aprepro/eagle_aprepro.txt:8]: Yes, same logic as above.

File: iea15mw/power_curve/submission_scripts/eagle-hpc-submit-power-curve.sh (hunk 4 above)
Comment (kevmoor) [reply iea15mw/power_curve/submission_scripts/eagle-hpc-submit-power-curve.sh]: Sounds good, it looks like I can't push to the branch to make up update though, maybe need to add me to a group/to this repo even though I'm now part of the Exawind github?

Comment (psakievich) [reply iea15mw/power_curve/submission_scripts/eagle-hpc-submit-power-curve.sh]: @kevmoor I'll try to see what is going on, but I'm running out of time. I leave tomorrow for a vacation. Pushing to PR branches is often restricted to only maintainers. In the meantime you can always open a PR against this branch.

**Pull request merged**